adoptedStyleSheets.js

```diff
@@ -26,27 +33,21 @@
     // custom elements
     createObserver(document.body);
 
+    // Document has loaded
     polyfillLoaded = true;
 
-    queue.forEach(({location, sheet}) => {
-      // sheet = Object.assign(new CSSStyleSheet(), sheet);
-      // console.log(sheet)
-      sheet.prototype = CSSStyleSheet.prototype;
-      const constructedStyleSheet = new CSSStyleSheet();
-      console.log(constructedStyleSheet)
-      for (const key in constructedStyleSheet) {
-        if (!sheet[key]) {
-          let prop = constructedStyleSheet[key];
-          if (typeof prop === 'function') {
-            prop = prop.bind(constructedStyleSheet);
-          }
-          sheet[key] = prop;
-        }
-      }
-      adoptStyleSheets(location)
+    // Move style elements created before document.body
+    // to the iframe along with future styles
+    deferredStyleSheets.forEach(nativeStyleSheet => {
```

Comment:
We may drop my approach solving the #12, but before it I would suggest to use `for..of` loop to be consistent.